internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -109,11 +125,15 @@ param (
     [bool]$insecure = $false,
     [string]$collector_version = "",
```

Comment:
nit: parameter naming convention error
https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/strongly-encouraged-development-guidelines?view=powershell-7.2#use-pascal-case-for-parameter-names

Reply:
Good to know. Perhaps it would be best to tackle this, along with some of the other items you all have mentioned when @jeffreyc-splunk gets back.